CHANGELOG.md

```diff
@@ -26,6 +26,16 @@
   Instead of it aggregate `near_peer_message_received_by_type_total` metric.
   For example, to get total rate of received messages use
   `sum(rate(near_peer_message_received_by_type_total{...}[5m]))`.
+* Few changes to `view_state` JSON RPC query:
+  - The requset has now an optional `include_proof` argument.  When set to
+    `true`, response’s `proof` will be populated.
+  - The `proof` within each value in `values` list of a `view_state` response is
+    now deprecated and will be removed in the future.  Client code should ignore
+    the field.
```

Comment:
Could we link some docs explaining how to verify the proof?

Reply:
Those docs currently don’t exist.  It is on my TODO to add them.  It’s possible that this might need to go into nomicon.

Reply:
Process-wise, I would slightly worrying about merging this without docs: docs seem like a phase where we can easily discover some design issues or what not. 

I don't think this is blocking, but I would prefer if we did docs before "stabilizing" things.